src/components/icon/icons/arrow-double.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="8" height="14" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M6.802 8.905a.7.7 0 01.991.991l-3.298 3.299a.7.7 0 01-.991 0L.205 9.896a.7.7 0 01.991-.99l2.716 2.714a.124.124 0 00.175 0l2.715-2.715zM3.504.205a.7.7 0 01.99 0l3.299 3.299a.7.7 0 11-.991.99L4.087 1.78a.124.124 0 00-.175 0L1.196 4.495a.7.7 0 11-.99-.991L3.503.205z" fill="gray"/>
```

Comment:
I think it would be more mobile to use the `fill="currentColor"` and define the color in the parent element